tests/keysignature_tests.ts

```diff
@@ -29,12 +29,18 @@ const KeySignatureTests = {
   },
 };
 
+function getWidth(code: number) {
+  const el = new Element();
+  el.setText(String.fromCharCode(code));
```

Comment:
I think in our examples, we should always use String.fromCodePoint() unless there is a good performance related reason to use fromCharCode(). The newer method handles all unicode code points properly.

Reply:
Agree but changed to unicode value, which should be the preferred method.